Documentation/openshift.md

```diff
@@ -3,11 +3,22 @@ title: OpenShift
 weight: 1700
 indent: true
 ---
+{% assign url = page.url | split: '/' %}
+{% assign currentVersion = url[3] %}
+{% if currentVersion != 'master' %}
+{% assign branchName = currentVersion | replace: 'v', '' | prepend: 'release-' %}
+{% else %}
+{% assign branchName = currentVersion %}
+{% endif %}
 
 # OpenShift
 
 [OpenShift](https://www.openshift.com/) adds a number of security and other enhancements to Kubernetes. In particular, [security context constraints](https://blog.openshift.com/understanding-service-accounts-sccs/) allow the cluster admin to define exactly which permissions are allowed to pods running in the cluster. You will need to define those permissions that allow the Rook pods to run.
 
+The settings for Rook in OpenShift are described below, and are also included in the [example yaml files](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph):
+- `operator-openshift.yaml`: Creates the scc.yaml before starting the operator deployment
+- `object-openshift.yaml`: Defines the rgw port in a valid range for OpenShift
```

Comment:
This section mentions some new YAML files, and we still run `oc create -f scc.yaml` later on. Are we supposed to do anything with these `*-openshift.yaml` files? What does "Creates the scc.yaml" mean?

Reply:
I'll clarify this. Basically, operator-openshift.yaml will include all of the settings needed to get the operator going, rather than needing to dig through this doc and create multiple yaml files.